src/TouristInfoCards.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+
+const TouristInfoCards = () => {
```

Comment:
Looking at https://cyf-yunusfirat-hotel-react.netlify.app/, I can see that the cards are all stacked up on top of each other. How could you fix this?

Reply:
I used bootstrap d-flex.